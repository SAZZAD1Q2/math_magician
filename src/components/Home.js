import React from 'react';
import styles from '../style/Home.module.css';

function Home() {
  return (
    <>
      <div className={styles.welcome}>
        <h1>welcome to My home page</h1>
        <p>
          The Math Magician App is a groundbreaking educational tool.
        </p>
        <p>
          One of the key features of the Math Magician App is its Quality.

        </p>
      </div>

    </>
  );
}

export default Home;
