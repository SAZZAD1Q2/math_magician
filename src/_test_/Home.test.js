import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Home', () => {
  test('should render the component', () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
