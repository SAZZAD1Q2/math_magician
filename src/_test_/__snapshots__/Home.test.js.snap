// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render the component 1`] = `
<div
  className="welcome"
>
  <h1>
    welcome to My home page
  </h1>
  <p>
    The Math Magician App is a groundbreaking educational tool.
  </p>
  <p>
    One of the key features of the Math Magician App is its Quality.
  </p>
</div>
`;
